src/server.cc

```diff
@@ -35,13 +35,13 @@ std::string Server::name() const { return "rbprm"; }
 
 /// Start corba server
 void Server::startCorbaServer(const std::string& contextId, const std::string& contextKind) {
-  bool mThd = parent()->multiThread();
-  rbprmBuilder_ = new corba::Server<impl::RbprmBuilder>(0, NULL, mThd, "child");
-  rbprmBuilder_->implementation().setServer(this);
```

Comment:
Should we really remove this line ? After this PR I cannot create robot of access server_->problemSolver() , and by adding this line again it work correclty.

Reply:
@jmirabel

Reply:
You are right, this last line should be kept. Sorry for the mistake.